ref_builder/resources.py

```diff
@@ -330,6 +330,17 @@ def get_isolate_id_by_name(self, name: IsolateName) -> UUID4 | None:
 
         return None
 
+    def get_isolate_id_by_partial(self, id_part: str):
```

Comment:
I'd rather hold off designing this until we are ready to use it.
I imagine it would be more effective to do this search with SQLite. Then you can have an OTU-agnostic command like:
```
ref-builder isolate get ghjs782l
```